DESCRIPTION

```diff
@@ -1,20 +1,25 @@
 Package: fledge
-Title: Smoother change tracking and versioning for R packages
+Title: Smoother Change Tracking and Versioning for R Packages
 Version: 0.0.5.9004
 Date: 2021-11-21
 Authors@R: 
     c(person(given = "Kirill",
              family = "M\u00fcller",
-             role = c("aut", "cre"),
+             role = c("aut", "cre", "cph"),
              email = "krlmlr+r@mailbox.org",
              comment = c(ORCID = "0000-0002-1416-3412")),
       person(given = "Patrick",
              family = "Schratz",
              role = "aut",
              email = "patrick.schratz@gmail.com",
-             comment = c(ORCID = "0000-0003-0748-6624")))
-Description: Streamlines the process of updating changelogs
-    (NEWS.md)
+             comment = c(ORCID = "0000-0003-0748-6624")),
+      person(given = "Maëlle", 
+             family = "Salmon",
+             role = "ctb",
+             email = "msmaellesalmon@gmail.com",
+             comment = c(ORCID = "0000-0002-2815-0399")))
+Description: Streamlines the process of updating 'changelogs'
```

Comment:
Can you please use a technique similar to RPostgres to get rid of the 'quotes'?

https://github.com/r-dbi/RPostgres/tree/main/.aspell

Reply:
Thanks to @jimhester!